Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -311,18 +305,16 @@ class AtlasStepper {
   /// @param [in] boundParams Parameters in bound parametrisation
   /// @param [in] cov Covariance matrix
   /// @param [in] surface Reset state will be on this surface
-  /// @param [in] navDir Navigation direction
   /// @param [in] stepSize Step size
   void resetState(
       State& state, const BoundVector& boundParams, const BoundSymMatrix& cov,
-      const Surface& surface, const Direction navDir = Direction::Forward,
+      const Surface& surface,
```

Comment:
We do have cases where the direction was changed from inside the propagation, see below.
Should this not reset the options in this case?

Reply:
since the stepper state property is gone now I think it is not the steppers responsibility anymore to change the direction

Reply:
Right, it's not at all inside the stepper state anymore. So the e.g. CKF has to do that itself now. Fair enough.